### DISCUSSION TITLE
求助大佬，关于禁用窗体最大化按钮

### DISCUSSION THREAD

**Discussion by oongvx** [tw93/Pake#515] [Q&A]
//禁止使用最大化按钮
this.MaximizeBox = false;

想设置窗口最大化按钮不可用，请问这串代码该加哪个配置文件里啊，代码结构说明找了一圈，还是没发现，万分感谢！

**Comment by oongvx** [comment]
解决了，在pake.json中，把"resizable": true,改成 "resizable": false,

**Reply by tw93** [reply]
哈哈哈哈 可以的
